src/scripts/main.js

```diff
@@ -1,3 +1,187 @@
+  });
+
+  newSelect.name = nameForm.toLowerCase();
+  newSelect.dataset.qa = nameForm.toLowerCase();
+  newLabel.append(newSelect);
+
+  return newLabel;
+}
+
+function sortForTable(sortTable, column) {
+  const tbody = sortTable.querySelector('tbody');
+
+  return Array.from(tbody.children).sort((row1, row2) => {
+    const value1 = row1.children[column].textContent;
+    const value2 = row2.children[column].textContent;
+
+    if (!/[0-9]/.test(value1)) {
+      return value1.localeCompare(value2);
+    } else if (/[0-9]/.test(value1)) {
+      return value1.replace(/\D/g, '') - value2.replace(/\D/g, '');
+    }
+  });
+}
+
+function createMessage(typeNotification, info) {
+  const notification = document.createElement('div');
+  const titleNotification = document.createElement('h2');
+  const textNotification = document.createElement('p');
+  const oldMessage = document.querySelector('.notification');
+
+  if (oldMessage) {
+    oldMessage.remove();
+  }
+
+  notification.dataset.qa = 'notification';
+  titleNotification.classList.add('title');
+
+  if (typeNotification === 'error') {
+    notification.classList.add('notification', 'error');
+    titleNotification.innerText = 'Incorrect data entered.';
+
+    if (info === 'age') {
+      textNotification.innerText = `Please check the ${info}. Minimum age 18, maximum 90.`;
+    } else {
+      textNotification.innerText = `Please check the ${info} and make corrections.`;
+    }
+  } else if (typeNotification === 'success') {
+    notification.classList.add('notification', 'success');
+    titleNotification.innerText = 'Success!';
+    textNotification.innerText = 'Data added successfully';
+  }
+
+  notification.append(titleNotification, textNotification);
+  document.body.append(notification);
+
+  window.setTimeout(() => notification.remove(), 3000);
+}
+
+function validationSalary(value) {
+  const clearNum = value.replace(/\D/g, '');
+  const formatedValue = parseFloat(clearNum).toLocaleString('en-US');
+
+  return '$' + formatedValue;
+}
+
+mainTable.tHead.addEventListener('click', function (eventData) {
+  if (!eventData.target.tagName === 'TH') {
+    return false;
+  }
+
+  const sortedElement = sortForTable(
+    this.parentElement,
+    eventData.target.cellIndex,
+  );
+
+  if (Event.sortBy !== eventData.target.textContent) {
+    Event.sortBy = eventData.target.textContent;
+    this.nextElementSibling.append(...sortedElement);
+  } else if (Event.sortBy === eventData.target.textContent) {
+    this.nextElementSibling.append(...sortedElement.reverse());
+    Event.sortBy = null;
+  }
+});
+
+mainTable.tBodies[0].addEventListener('click', function (eventData) {
+  const row = eventData.target.closest('tr');
+  const activeRow = this.querySelector('tr.active');
+
+  if (activeRow && activeRow !== row) {
+    activeRow.classList.remove('active');
+  }
+
+  if (row) {
+    row.classList.toggle('active');
+  }
+});
+
+newForm.addEventListener('submit', (evetntData) => {
```

Comment:
There is a typo in the parameter name `evetntData`. It should be `eventData` to maintain consistency and avoid confusion.